e2e/pages/enterApplicant/enterApplicant.js

```diff
@@ -0,0 +1,46 @@
+const applicant = require('../../fixtures/applicant.js');
+const solicitor = require('../../fixtures/solicitor.js');
+
+const I = actor();
+
+module.exports = {
+
+	fields: {
+		applicant: {
+			name: '#enterApplicant_name',
+			nameOfPersonToContact: '#enterApplicant_personToContact',
+			jobTitle: '#enterApplicant_jobTitle',
+			address: '#enterApplicant_address',
+			mobileNumber: '#enterApplicant_mobile',
+			telephoneNumber: '#enterApplicant_telephone',
+			email: '#enterApplicant_email',
+		},
+		solicitor: {
+			name: '#enterApplicant_solicitorName',
+			mobileNumber: '#enterApplicant_solicitorMobile',
+			telephoneNumber: '#enterApplicant_solicitorTelephone',
+			email: '#enterApplicant_solicitorEmail',
+			dx: '#enterApplicant_solicitorDx',
+			reference: '#enterApplicant_solicitorReference',
+		}
+	},
+
+	enterApplicantDetails() {
```

Comment:
To make it more configurable I would suggest passing applicant in method args. This way here we get:

```
enterApplicantDetails(applicant) {
  ...
}
```

And in test file we have:

```
const applicant = require('../../fixtures/applicant.js');
```

Reply:
Good spot 👍